src/A2A/Models/Part.cs

```diff
@@ -1,17 +1,55 @@
 using System.Text.Json;
 using System.Text.Json.Serialization;
+using System.Text.Json.Serialization.Metadata;
 
 namespace A2A;
 
+/// <summary>
+/// Defines the set of Part kinds used as the 'kind' discriminator in serialized payloads.
+/// </summary>
+/// <remarks>
+/// Values are serialized as lowercase kebab-case strings via <see cref="KebabCaseLowerJsonStringEnumConverter{TEnum}"/>.
+/// </remarks>
+[JsonConverter(typeof(KebabCaseLowerJsonStringEnumConverter<PartKind>))]
+public enum PartKind
```

Comment:
NIT: Generally in objects which are deserialized into (deserialization model) I am trying to follow best practices of having `default` enum values as `invalid` or `unknown` so if for any reason this is actually not assigned from deserialized data (common bugs) it will be easily detectable.
Please consider to add `Unknown` as 1st and maybe `Count` as last if we still decide to have mapping array from its values.